osu.Game.Tests/Resources/TestResources.cs

```diff
@@ -20,10 +21,10 @@ public static string GetTestBeatmapForImport(bool virtualTrack = false)
             var temp = Path.GetTempFileName() + ".osz";
 
             using (var stream = GetTestBeatmapStream(virtualTrack))
-            using (var newFile = File.Create(temp))
+            using (var newFile = RuntimeInfo.StartupStorage.GetStream(temp, FileAccess.Write))
```

Comment:
A few more usages of `StartupStorage` that should be removed.

Reply:
Whoops, will remove